BurakOlgun/hw4/resources/views/frontend/views/components/RestaurantComponents/RestaurantList/restaurantList.html

```diff
@@ -27,19 +27,14 @@
 <button class="form-control btn btn-info" ng-click="newRestaurantShower()">Add New Restaurant</button>
 <div class="form-control" ng-show="tab5">
     <label class="col-form-label-sm">Name</label>
-    <input class="form-control" type="text" ng-model="restaurantName"  required>
+    <input class="form-control" type="text" ng-model="name"  required>
```

Comment:
aslinda bu inputlar newRestaurant objesinin özellikleri. bu input modellerini şoyle yapsan 
`ng-model="newRestaurant.restauranName"` böylece aşağida push yaparken üç ayri parametre yollamak yerine sadece newRestaurant yollarsin hatta daha önce yaptiğin gibi inline olarak `newRestaurantDatas.push(newRestaurant)` yazabilirsin.